R/ui.R

```diff
@@ -70,7 +86,7 @@ ui_code_block <- function(x, copy = interactive(), .envir = parent.frame()) {
   x <- glue(x, .envir = .envir)
 
   block <- indent(x, "  ")
-  block <- crayon::make_style("darkgrey")(block)
+  block <- crayon::silver(block)
```

Comment:
If I can't have a decent red, then you can't have dark grey 😈

More seriously, is there some principle that makes this use of `crayon::make_style()` acceptable but not others?

Reply:
No 😐